src/chain_sync/tipset_syncer.rs

```diff
@@ -771,7 +772,7 @@ async fn sync_tipset_range<DB: Blockstore + Sync + Send + 'static>(
         &bad_block_cache,
         parent_tipsets,
         &genesis,
-        InvalidBlockStrategy::Strict,
+        InvalidBlockStrategy::Forgiving,
```

Comment:
What are the consequences of moving from `Strict` to `Forgiving`?

Reply:
Below is the only usage of `InvalidBlockStrategy::Strict` I found. It uses a bad block cache to bypass validating known bad blocks, but I keep getting `chain contained [CID]` during sync in Strict mode. To my understanding, using `forgiving` mdoe is disabling a perf trick instead of  changing the logic
```rust
// Only do bad block accounting if the function was called with
                // `is_strict` = true
                if let InvalidBlockStrategy::Strict = invalid_block_strategy {
                    match &why {
                        TipsetRangeSyncerError::TimeTravellingBlock(_, _)
                        | TipsetRangeSyncerError::TipsetParentNotFound(_) => (),
                        why => {
                            bad_block_cache.put(cid, why.to_string());
                        }
                    }
                }
```